cron/alerts/main_test.go

```diff
@@ -40,9 +40,5 @@ func TestGetCFRecords(t *testing.T) {
 		return
 	}
 
-	_, err := getCFRecords(config)
-	if err != nil {
-		t.Errorf("Failed getting results: %v", err)
-		return
-	}
+	_ = getCFRecords(config)
```

Comment:
Hopefully test doesn't send emails.

Reply:
The test is disabled and only there for local development.